src/styles/animation.styl

```diff
@@ -46,8 +46,16 @@
       width: 100vw;
     }
 
-    .logoSvg{
-      font-size: 210px;
+    .logoText {
+      color: black;
+      text-shadow: stroke(5, $titlesColor);
+      letter-spacing: .18em;
+      font-size: 200px;
+      line-height: 1em;
+
+      div {
+        white-space: nowrap;
```

Comment:
Check this for long lines:

```suggestion
        white-space: nowrap;
        margin-left: -100%;
        margin-right: -100%;
```

![image](https://user-images.githubusercontent.com/6919329/49341600-d2329f80-f636-11e8-84e5-2cbf06cc0054.png)

Reply:
Done: f4eb8aca9f5dc4ccf1e9eaa3a079efb2c7ae235a

Reply:
👌